src/main/java/app/AppBuilder.java

```diff
@@ -72,11 +72,6 @@
 public class AppBuilder {
```

Comment:
I see that you have deleted these imports because they are not being used by the methods in the app builder however, those changes were made by Matteo and I to temporarily fix compilation problems might require changes to be updated for everyone's usecases. I would suggest erasing this commit till those methods are updated.